concourse/scripts/ssh.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -eu
+
+key=/tmp/concourse_id_rsa.$RANDOM
+trap "rm -f $key" SIGINT SIGTERM EXIT
+
+eval $(make dev showenv | grep CONCOURSE_IP=)
+
+aws s3 cp "s3://${DEPLOY_ENV}-state/concourse_id_rsa" $key && chmod 400 $key
+
+echo
+aws s3 cp "s3://${DEPLOY_ENV}-state/generated-concourse-secrets.yml" - | \
+ruby -ryaml -e 'puts "\Sudo password is " + YAML.load(STDIN)["secrets"]["concourse_vcap_password_orig"]' 
+echo
+
+ssh -i concourse_id_rsa -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null vcap@$CONCOURSE_IP
```

Comment:
I think this needs the path to the keyfile here. At present, this doesn't work for me, I get the following:
```
$ make dev ssh_concourse
download: s3://alext-state/concourse_id_rsa to ../../../../tmp/concourse_id_rsa.4674

Sudo password is <REDACTED>

Warning: Identity file concourse_id_rsa not accessible: No such file or directory.
Warning: Permanently added '<REDACTED_IP>' (ECDSA) to the list of known hosts.
Unauthorized use is strictly prohibited. All access and activity
is subject to logging and monitoring.
Permission denied (publickey).
make: *** [ssh_concourse] Error 255
```